jupyterhub/notebook-images/overlays/cuda-11.0.3/cuda-ubi8-build-chain.yaml

```diff
@@ -9,143 +9,201 @@ items:
       local: true
     tags:
     - name: latest
+      annotations: null
       from:
         kind: DockerImage
-        name: "nvcr.io/nvidia/cuda:$(cuda_version)-cudnn8-devel-ubi8"
+        name: 'nvcr.io/nvidia/cuda:$(cuda_version)-cudnn8-devel-ubi8'
+      generation: 2
```

Comment:
Why are we specifying `generation: 1` for this one, and `generation: 1` for all the others?

Reply:
the generation 2 is specified here.
as 
generation 1 would be for tag creation
and generation 2 would be for image pull from the defined registry.